### PR TITLE
.trivyignore: ignore cve

### PR DESCRIPTION
**Pull request by sam-heilbron**
# Description

Update `.trivyignore` contents

# Context
https://github.com/solo-io/solo-projects/issues/7157#issuecomment-2463252858

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: .trivyignore [hunk 1]
```diff
@@ -62,3 +62,9 @@ CVE-2024-45258
 # Therefore we include this entry for now and should remove it once 1.14 is no longer an LTS branch
 CVE-2024-27289
 CVE-2024-27304
+
+# https://github.com/advisories/GHSA-5fhx-39r8-3jwh
+# This is resolved in versions of Gloo Gateway that rely on Go1.22 and above (1.17, 1.18)
+# For ealier versions of Gloo Gateway, we confirmed that the vulnerability is not exploitable
```
Comment (jenshu) [inline .trivyignore:68]:
```suggestion
# For earlier versions of Gloo Gateway, we confirmed that the vulnerability is not exploitable
```

File: .trivyignore [hunk 2]
```diff
@@ -62,3 +62,9 @@ CVE-2024-45258
...
+# and captured our findings here: https://github.com/solo-io/solo-projects/issues/7157#issuecomment-2463252858
```
Comment (jenshu) [inline .trivyignore:69]: nit: ideally shouldn't link to a private repo

**Review by jenshu — APPROVED** [review]

File: .trivyignore (hunk 2 above)
Comment (sam-heilbron) [reply .trivyignore:69]: I agree, though it holds all the details so I wanted to make sure it was discoverable. Do you think we have an alternative?

Comment (jenshu) [reply .trivyignore:69]: maybe if we had a public GH issue. but i'm fine with this for now

**Review by jenshu — APPROVED** [review]

**Review by andy-fong — APPROVED** [review]

**Comment by sam-heilbron** [comment]
I encountered a test flake and created https://github.com/k8sgateway/k8sgateway/issues/10375

**Pull request merged**